README.md

```diff
@@ -122,23 +122,23 @@ These system requirements must be met for using the compiled binaries.
     `update-locale LANG=en_US.UTF-8`
   * Follow similar procedure to configure other locales on other platforms.
   
-#### Default CPU
+#### CPU
 * ONNX Runtime binaries in the CPU packages use OpenMP and depend on the library being available at runtime in the
 system.
   * For Windows, **OpenMP** support comes as part of VC runtime. It is also available as redist packages:
     [vc_redist.x64.exe](https://aka.ms/vs/16/release/vc_redist.x64.exe) and [vc_redist.x86.exe](https://aka.ms/vs/16/release/vc_redist.x86.exe)
   * For Linux, the system must have **libgomp.so.1** which can be installed using `apt-get install libgomp1`.
 
-#### Default GPU (CUDA)
-* The default GPU build requires CUDA runtime libraries being installed on the system:
-	 * Version: **CUDA 10.1** and **cuDNN 7.6.5**
-* Version dependencies from older ONNX Runtime releases can be found in [prior release notes](https://github.com/microsoft/onnxruntime/releases).
+#### GPU
+* GPU builds require CUDA runtime libraries being installed on the system:
+	 * Version: **CUDA 10.0** and **cuDNN 7.6**
+  * Older ONNX Runtime releases used **CUDA 9.1** and **cuDNN 7.1** - please refer to [prior release notes](https://github.com/microsoft/onnxruntime/releases) for more details.
 
-#### Other Execution Providers
-* For requirements and dependencies of other build options, see detailed build instructions on the [BUILD.md](./BUILD.md#additional-build-instructions) page.
 ***
 # Usage
-Please see [Samples and Tutorials](./samples) for examples.
+## [Samples and Tutorials](./samples)
+
+## [Frequently Asked Questions and Troubleshooting](./docs/faq.md)
```

Comment:
need to preserve case.
should be FAQ.md otherwise link is broken.
can we check all the links before submitting?